docs/python/conda.md

```diff
@@ -0,0 +1,142 @@
+# Use Conda to Manage Your Own Python Environments
+
+Managing software dependencies can be one of the most annoying thing in scientific computing, 
+especially when dealing with complex systems with multiple packages and libraries. 
+This is where conda comes in - a package management system that simplifies the task of installing, configuring, 
+and managing software packages and their dependencies.
+
+BEAR systems does offer a default Python environment, but it must be loaded beforehand, 
+and it does not allow you to install additional packages that are not already provided.
+After creating your own Conda environment, you don't have to load them everytime. 
+Submitting cluster jobs becomes straightforward with simple scripts, as demonstrated in this example:
+
+```bash
+#!/bin/bash
+set -e
+# SBATCH settings
+python your_script.py
+```
+
+!!! note BEAR [do not currently recommend using conda for installing python packages](https://docs.bear.bham.ac.uk/bluebear/software/self_installs_python/#tips). This is due to possible performance issues when installing packages with conda. BEAR recommend that you get in touch with them prior via an IT-ticket to performing intensive analyses using conda envinroments. 
+
+If this seems interesting to you or you need to use a custom enviroment, let's get started!
+
+## What is conda
+
+Conda is a platform-agnostic package management system that can be used to install and manage software packages and their dependencies.
+It is designed to work with multiple programming languages, including Python, R, and others.
+
+Advantages of using conda:
+* Build your own python env when you don't have `sudo` access
```

Comment:
```suggestion

* Build your own python env when you don't have `sudo` access
```

Reply:
I have fixed this. I didn't know Markdown had these formatting preferences before.

Reply:
Thanks - it's an additional formatting checker for Markdown, I may make it optional in future as its quite strict...